### DISCUSSION TITLE
Tasks stop when conductor server dies

### DISCUSSION THREAD

**Discussion by AdityaMishraGit** [Netflix/conductor#3235] [Q&A]
Hi,  We are using netflix conductor with postgres database ,  but the tasks are getting stuck if the conductor server dies. We have a requirement where the task should auto start once we restart the conductor server. Is there a way to do so or are we missing something?

**Comment by v1r3n** [comment]
Hi @AdityaMishraGit do the task remain IN_PROGRESS status or SCHEDULED status after the workflow restarts?

**Reply by AdityaMishraGit** [reply]
Hi , thanks for replying. It remains on SCHEDULED state

**Reply by v1r3n** [reply]
Can you check the workflow sweeper is running?  It is enabled by default unless configured to disable.  The following property controls it
```
conductor.workflow-reconciler.enabled
```

**Reply by AdityaMishraGit** [reply]
The property is enabled . I was able to repair the workflows stuck by the verifyAndRepair api end point , thanks to the hint.

I guess I will need to write a schedular to repair my stuck workflows

**Reply by aravindanr** [reply]
fwiw, enabling the property `conductor.workflow-repair-service.enabled` runs the `verifyAndRepair` on a schedule.
